packages/lodestar-utils/src/json.ts

```diff
@@ -2,26 +2,29 @@ import {Json, toHexString} from "@chainsafe/ssz";
 import {LodestarError} from "./errors";
 
 export function toJson(arg: unknown): Json {
-  switch (typeof arg) {
-    case "bigint":
-    case "symbol":
-    case "function":
-      return arg.toString();
-
-    case "object":
-      if (arg === null) return "null";
-      if (arg instanceof Uint8Array) return toHexString(arg);
-      if (arg instanceof LodestarError) return toJson(arg.toObject());
-      if (arg instanceof Error) return toJson(errorToObject(arg));
-      return arg as Json;
+  try {
```

Comment:
The `toJson` helper was fixed in https://github.com/ChainSafe/lodestar/pull/1669. You can rebase to master and use its solution if you believe its sufficient